EXT_SpeciesType.md

```diff
@@ -0,0 +1,95 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+openPMD extension ID: `4`
+
+
+Introduction
+------------
+
+This convention is for standardizing the names of particle species, e.g. in
+particle physics.
+
+
+Additional Record Attribute
+---------------------------
+
+The following additional attribute for openPMD `records` is defined in this
+extension:
+
+- `SpeciesType`
+  - type: *(string)*
+  - scope: *optional*
+  - description: particle species in this record. If there are multiple
+                 species to be specified, they can be specified using a
+                 semicolon separated list.
+  - allowed values:
+    - *see the lists below* and additionally
+    - `other` if none of the ones below applies, user are free to append a
+      free text after a colon, e.g. `other:neutralino` or `other:cherry`
+  - examples:
+    - `electron` (e.g. on an electron `particle record` or an electron
+                  density `mesh record`)
+    - `electron;proton;#12C` (e.g. on a `mesh record` for a plasma's
+                              local charge density)
+    - `other:apple;other:orange` (for a `record` mixing apples & oranges)
+
+This attribute can be used with any `record` (including `mesh records`).
+
+### Elementary Particles
+
+Namings for fundamental fermions and their anti-matter particles.
+
+Quarks:
+  - `up` (`anti-up`)
+  - `down` (`anti-down`)
+  - `charm` (`anti-charm`)
+  - `strange` (`anti-strange`)
+  - `top` (`anti-top`)
+  - `bottom` (`anti-bottom`)
+
+Leptons:
+  - `electron` (`positron`)
+  - `electron-neutrino` (`anti-electron-neutrino`)
```

Comment:
anti-neutrinos: who knows, maybe they turn out to be the same particles. but until now it seems that calling them anti-neutrinos is common